Square frontend

TiiLeaf opened the pull request.

rahulshamkuwar reviewed (changes requested):
Still need to implement navbar and alertbox.

TiiLeaf commented:
Added a dropdown to the navbar, the square page navbar is now the navbar for every page, re-colored dropdowns and buttons, added partials to square.ejs, standardized the way alerts are displayed on square.ejs, refactored the logic for chat history.
